.gitignore

```diff
@@ -1,2 +1,5 @@
 Gemfile.lock
-tmp
\ No newline at end of file
+tmp
+/gemfiles/*.lock
+.tool-versions
+.rspec
```

Comment:
did you mean to ignore `.rspec`?

Reply:
I did this thinking I would be the maintainer originally. I have a bring-your-own environment philosophy, so everyone would get to define their own `.rspec` config. Up to y'all to keep it or not.

Reply:
Yes @packrat386! Habit from Stitch Fix. We always ignore the `.rspec` file as it usually only stores display options (which is the case here too). I see it's still tracked though, so I'll remove it from git

Reply:
I left it for that same reason @adsteel 😄